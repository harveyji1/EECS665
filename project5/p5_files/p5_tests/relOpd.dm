student: class{
    myStudent: int;
};

fn: () int{
    hello: bool = true;
    if(hello<1){
        give hello;
    }
    myStudent: student;
    bye: bool = myStudent < 1;
}

main: () void{
    fn();
}