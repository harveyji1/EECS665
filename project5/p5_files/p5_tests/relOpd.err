FATAL [7,8]-[7,13]: Relational operator applied to non-numeric operand
Type Analysis Failed
