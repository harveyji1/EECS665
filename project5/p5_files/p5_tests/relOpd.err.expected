FATAL [3,8]-[3,13]: Relational operator applied to non-numeric operand
Type Analysis Failed